src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
Creating a new instance of `Random` every time the method is called is inefficient. Instead, you should create a single `Random` instance as a class field and reuse it.

Reply:
created private Random random
created a constructor without parameters

Reply:
not fixed